--- nabou-2.4/Makefile.in.orig	2004-08-30 23:10:35.000000000 +0200
+++ nabou-2.4/Makefile.in	2004-11-19 15:56:41.000000000 +0100
@@ -50,9 +50,9 @@
 
 PREFIX  = @prefix@
 VERSION = @VERSION@
-MAN     = $(PREFIX)/man/man1
+MAN     = $(PREFIX)/share/man/man1
 SBIN    = $(PREFIX)/sbin
-DOC     = $(PREFIX)/doc/nabou-$(VERSION)
+DOC     = $(PREFIX)/share/doc/nabou-$(VERSION)
 ETC     = @sysconfdir@
 DB      = @localstatedir@
 DISTCFG = nabourc.dist
@@ -86,25 +86,25 @@
 
 
 install:
-	$(install) -d -m 755 -o $(OWNER) -g $(GROUP) $(PREFIX) $(MAN) $(SBIN) $(DOC)
-	$(install) -d -m $(D_PERM) -o $(OWNER) -g $(GROUP) $(ETC)
-	$(install) -d -m $(N_PERM) -o $(OWNER) -g $(GROUP) $(DB)
+	$(install) -d -m 755 $(DESTDIR)$(PREFIX) $(DESTDIR)$(MAN) $(DESTDIR)$(SBIN) $(DESTDIR)$(DOC)
+	$(install) -d -m $(D_PERM) $(DESTDIR)$(ETC)
+	$(install) -d -m $(N_PERM) $(DESTDIR)$(DB)
 
-	$(install) -m $(N_PERM) -o $(OWNER) -g $(GROUP) bin/nabou $(SBIN)/
+	$(install) -m $(N_PERM) bin/nabou $(DESTDIR)$(SBIN)/
 
 ifeq 	($(GOTCFG), $(ETC)/nabourc)
 	@echo "*** Existing config '$(ETC)/nabourc' will be kept untouched. ***"
 else
 	echo $(GOTCFG)
-	$(install) -m $(C_PERM) -o $(OWNER) -g $(GROUP) $(DISTCFG) $(ETC)/nabourc
+	$(install) -m $(C_PERM) $(DISTCFG) $(DESTDIR)$(ETC)/nabourc
 endif
 
 
-	$(install) -m 644 -o $(OWNER) -g $(GROUP) man/* $(MAN)/
+	$(install) -m 644 man/* $(DESTDIR)$(MAN)/
 
-	$(install) -d -m 755 -o $(OWNER) -g $(GROUP) $(DOC)/sample_configs
-	$(install) -m 644 -o $(OWNER) -g $(GROUP) sample_configs/* $(DOC)/sample_configs/
-	$(install) -m 644 -o $(OWNER) -g $(GROUP) $(DOCS) $(DOC)/
+	$(install) -d -m 755 $(DESTDIR)$(DOC)/sample_configs
+	$(install) -m 644 sample_configs/* $(DESTDIR)$(DOC)/sample_configs/
+	$(install) -m 644 $(DOCS) $(DESTDIR)$(DOC)/
 
 	@echo
 	@echo "***"
